.gitignore

```diff
@@ -11,6 +11,7 @@ node_modules
 # production
 /build
 dist
```

Comment:
Think with `.next` as build dir we can remove `dist`.

Reply:
server uses `dist` as output dir tho

Reply:
🤦  u right